src/components/selectors/MimicSelector.tsx

```diff
@@ -50,13 +50,19 @@ const MimicSelector: React.FC<MimicSelectorProps> = ({ MimicData, mimicNumber, o
     return value != null && value.length > 0;
   };
 
+  React.useEffect(() => {
+    let numEmptyField = Object.keys(MimicData).length - 1; // minus gameId in MimicData
+    for (const [key, value] of Object.entries(MimicData)) {
+      if (key === 'gameId') continue;
+      if (isFieldValid(value as string)) numEmptyField -= 1;
+    }
+    if (numEmptyField < Object.keys(MimicData).length - 1) setIsError(true);
+  }, []);
+
   const nextPage = () => {
     save().catch(console.error);
-    if (isValid()) {
-      onNext?.();
-    } else {
-      setIsError(true);
-    }
+    if (!isValid()) setIsError(true);
+    onNext?.();
```

Comment:
Setting the `isError` variable here is not useful anymore because the user navigates to another page. (See my suggestion above.)

Reply:
I deleted these lines.